python_modules/dagster/dagster/_core/definitions/asset_reconciliation_sensor.py

```diff
@@ -316,30 +319,28 @@ def from_serialized(cls, cursor: str, asset_graph: AssetGraph) -> "AssetReconcil
             check.invariant(len(data) in [3, 4], "Invalid serialized cursor")
             (
                 latest_storage_id,
-                serialized_materialized_or_requested_root_asset_keys,
-                serialized_materialized_or_requested_root_partitions_by_asset_key,
+                serialized_handled_root_asset_keys,
+                serialized_handled_root_partitions_by_asset_key,
             ) = data[:3]
 
             evaluation_id = data[3] if len(data) == 4 else 0
             serialized_last_observe_request_timestamp_by_asset_key = {}
         else:
             latest_storage_id = data["latest_storage_id"]
-            serialized_materialized_or_requested_root_asset_keys = data[
-                "materialized_or_requested_root_asset_keys"
-            ]
-            serialized_materialized_or_requested_root_partitions_by_asset_key = data[
-                "materialized_or_requested_root_partitions_by_asset_key"
+            serialized_handled_root_asset_keys = data["handled_root_asset_keys"]
```

Comment:
is the idea that we don't need to handle backcompat for the old names because we haven't released since we added them?

Reply:
exactly yep